images/pdc-update/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "pdc-update"
+version = "0.1.0"
+edition = "2021"
```

Comment:
```suggestion
edition = "2024"
```

Reply:
The 2024 edition has not been released yet